.github/workflows/audit_service_tags.yml

```diff
@@ -61,8 +61,13 @@ jobs:
           cache-to: type=gha,mode=max
 
       - name: Setup Database
-        run: |
-             docker compose -f docker-compose.test.yml run web bash \
+        uses: nick-fields/retry@7152eba30c6575329ac0576536151aca5a72780e # v3.0.0
+        with:
+          timeout_minutes: 20
+          retry_wait_seconds: 3 # Seconds
+          max_attempts: 3
+          command: |
```

Comment:
Do we also need to use the `retry_on` input to only retry after an error? I assumed that would be the default... but looking at [the docs](https://github.com/nick-fields/retry?tab=readme-ov-file#only-retry-after-error) maybe we need to specify it?

Reply:
It says it's optional and the default is to retry on error *and* timeout, which _seems_ sane to me. What are your thoughts though?

Reply:
oh yeah.. I see that now :)